functest/create_member_test.go

```diff
@@ -47,3 +47,18 @@ func TestCreateMemberWithBadKey(t *testing.T) {
 	require.NotNil(t, err)
 	require.Contains(t, err.Error(), fmt.Sprintf("problems with decoding. Key - %s", member.pubKey))
 }
+
+func TestCreateMembersWithSameName(t *testing.T) {
+	member, err := newUserWithKeys()
+	require.NoError(t, err)
+	member.ref = root.ref
+
+	addBurnAddresses(t)
+
+	_, err = retryableCreateMember(member, "contract.createMember", map[string]interface{}{}, false)
+	require.NoError(t, err)
+
+	_, err = retryableCreateMember(member, "contract.createMember", map[string]interface{}{}, false)
```

Comment:
Here use `signedRequest` because `member for this publicKey already exist` error is that we want to get, no need to retry `createMember` call